docs/src/main/paradox/bootstrap.md

```diff
@@ -210,6 +210,26 @@ is strictly defined and is as follows:
   to be using the Bootstrap mechanism. 
 @@@
 
+## Recommended Configuration
+
+When using the bootstrap module, there are some underlying Akka Cluster settings that should be specified to ensure
+that your deployment is robust.
+
+Since the target environments for this module are dynamic, that is, instances can come and go, failure needs to be dealt
+considered. The following configuration will result in your application being shut down after 40 seconds if it is unable to
```

Comment:
This sounds like a mistake, it should be either "failure needs to be considered" or "failure needs to be dealt with".

Reply:
Oops :blush: - https://github.com/akka/akka-management/pull/121